AdoNet.Tests.Console/Program.cs

```diff
@@ -185,6 +186,38 @@ static void Main(string[] args)
             };
 
             adoClient.SerializeAndWriteToFile(githubPipeline, "github-pipelines.yaml");
+
+            GitHubPipelineBuilder.CreateNewPipeline()
+                .SetName("Github")
+                .OnPush("master")
+                .OnPullRequest("master")
+                .AddJob("build", job => job
+                    .WithName("Build")
+                    .RunsOn(BuildMachines.WindowsLatest)
+                    .AddEnvironmentVariable("AzureClientId", "${{ secrets.AZURECLIENTID }}")
+                    .AddEnvironmentVariable("AzureTenantId", "${{ secrets.AZURETENANTID }}")
+                    .AddEnvironmentVariable("AzureClientSecret", "${{ secrets.AZURECLIENTSECRET }}")
+                    .AddEnvironmentVariable("AzureAdminName", "${{ secrets.AZUREADMINNAME }}")
+                    .AddEnvironmentVariable("AzureAdminAccess", "${{ secrets.AZUREADMINACCESS }}")
+                    //.AddEnvironmentVariables(new Dictionary<string, string>
```

Comment:
Remove commented section

Reply:
Yeah I am saving to an other file: `.SaveToFile("github-pipelines-2.yaml");`